app/docker/components/host-overview/host-overview-controller.js

```diff
@@ -0,0 +1,3 @@
+angular
```

Comment:
I don't see the point of adding an empty controller. Just remove it for now, if we need to tune the controller behavior later on, it will have to be introduced.